app/content/management/commands/migrate_members.py

```diff
@@ -0,0 +1,64 @@
+from django.core.management.base import BaseCommand
+from django.db.transaction import atomic
+
+from app.content.models import User
+from app.content.models.user import CLASS
+from app.group.models import Group, Membership
+
+STUDY_FULL = (
+    (1, "Dataingeniør"),
+    (2, "Digital forretningsutvikling"),
+    (3, "Digital infrastruktur og cybersikkerhet"),
+    (4, "Digital samhandling"),
+    (5, "Drift"),
+    (6, "Informasjonsbehandling"),
+)
+
+
+class Command(BaseCommand):
```

Comment:
Hva tenker du om å bruke `migrations.RunPython` istedenfor en command for å kjøre dette? Da kan det kjøres automatisk når vi merger både til dev og prod uten å måtte koble til databasen selv fra egen pc. Se `app\content\migrations\0036_events_add_cronjob_fields.py` for eksempel

Reply:
Denne skulle ikke med her men ja. Litt skummelt hvis det skjer noe krøll så må vi legge inne en fiks eller reverte med en gang for å få opp api-et igjen. Veldig digg at det skjer automatisk i prod og dev.

Reply:
Skal sjekke det ut jeg:)